packages/little_logger/little_logger.0.1.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis: "A tiny, little logger <3"
+maintainer: "Ryan Moore"
+authors: "Ryan Moore"
+license: "MIT"
+homepage: "https://github.com/mooreryan/little_logger"
+doc: "https://mooreryan.github.io/little_logger/"
+bug-reports: "https://github.com/mooreryan/little_logger/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "async" {dev}
+  "bisect_ppx" {dev}
+  "core" {>= "v0.12"}
+  "core_bench" {>= "v0.12" & dev}
+  "easy_logging" {dev}
+  "dolog" {dev}
+  "ocaml" {>= "4.08.0"}
+  "ppx_compare"
+  "ppx_inline_test" {dev}
+  "ppx_expect" {dev}
+  "ppx_sexp_conv" {dev}
+  "ppx_jane" {dev}
+  "re2" {dev}
```

Comment:
All the `{dev}` dependencies seem to be `{with-test}` instead:
```
#=== ERROR while compiling little_logger.0.1.0 ================================#
# context              2.0.9 | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/mooreryan/little_logger/archive/0.1.0.tar.gz)
# path                 ~/.opam/4.08/.opam-switch/build/little_logger.0.1.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p little_logger -j 47 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/little_logger-64-5f1023.env
# output-file          ~/.opam/log/little_logger-64-5f1023.out
### output ###
# File "test/async/dune", line 3, characters 35-40:
# 3 |  (libraries core little_logger re2 async)
#                                        ^^^^^
# Error: Library "async" not found.
# Hint: try:
#   dune external-lib-deps --missing --no-config --root . --ignore-promoted-rules --default-target @install --always-show-command-line --promote-install-files --release --only-packages little_logger -p little_logger --profile release -j 47 @install @runtest
# File "test/dune", line 3, characters 31-34:
# 3 |  (libraries core little_logger re2)
#                                    ^^^
# Error: Library "re2" not found.
# Hint: try:
#   dune external-lib-deps --missing --no-config --root . --ignore-promoted-rules --default-target @install --always-show-command-line --promote-install-files --release --only-packages little_logger -p little_logger --profile release -j 47 @install @runtest
```

Reply:
Good point, thank you!